spec/formatting.md

```diff
@@ -671,16 +672,26 @@ MAY rely on the introspection of its contents, or on other means.
 The directionality of the message as a whole is provided by the _formatting context_.
 
 If a formatted _expression_ itself contains spans with differing directionality,
-its formatter SHOULD isolate such parts to avoid
-negatively affecting the presentation of the overall formatted result.
-
-> For example, an implementation could provide a `:number` formatting function
-> which would always produce output matching the message's locale,
-> allowing for its formatted string representation to never need isolation.
+its formatter SHOULD perform any necessary processing, such as inserting controls or
+isolating such parts to ensure that the formatted value displays correctly in a plain text context.
+
+> For example, an implementation could provide a `:currency` formatting function
+> which inserts RLM, ALM, or LRM characters to properly display a currency symbol
+> next to a formatted number.
+> An example of this is formatting the value `1234.56` as the currency `AED`
+> in the `ar-AE` locale. The formatted value appears like this:
+> ```
+> ‎-1,234.56 د.إ.‏
+> ```
+> The code point sequence produced by the ICU4J `NumberFormat` function for this is:
+> **U+200F U+200E** U+002D U+0031 U+002C U+0032 U+0033 U+0034 U+002E U+0035 U+0036
+> U+00A0 U+062F U+002E U+0625 U+002E **U+200F**
+> Note that U+200F is RIGHT-TO-LEFT MARK and U+200E is LEFT-TO-RIGHT MARK.
 
 Implementations formatting messages as a concatenated string or a sequence of strings
 MUST provide one or more strategies for bidirectional isolation.
-One such strategy MUST behave as follows:
+One such strategy MUST be the **_Plain Text Bidirectional Isolation Strategy_**, which behaves
+as follows:
```

Comment:
This needs a snappier name; a single word (maybe max two) that can be used as an option value. I'm also not sure about "plain text", given that other strategies that apply more introspection could apply equivalently well to plain text.

Reply:
How about the new text's version: _Default Bidi Strategy_?

Reply:
`default` is a good name.